performance/venv.py

```diff
@@ -240,4 +240,4 @@ def create_virtualenv(python):
 def exec_in_virtualenv(options):
     venv_python = create_virtualenv(options.python)
     args = [venv_python, "-m", "performance"] + sys.argv[1:] + ["--inside-venv"]
-    os.execv(args[0], args)
+    run_cmd(args)
```

Comment:
> Any particular reason why?

It avoids to have two running processes. Using os.execv(), you only get one process. It might help to get more reliable benchmarks.

> That's what the return in the cli.py file does. I think the caller should do it explicitly.

Using os.execv(), the function never returns. So it's better to have the same behaviour if os.execv() or run_cmd() is used.